ci/docker/run/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y git build-essential  autoconf flex bison \
+                                         bc perl python3
+
+ARG REPO=https://github.com/verilator/verilator
+ARG SOURCE_COMMIT=master
+
+WORKDIR /tmp
+
+RUN git clone ${REPO} verilator && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)